packages/@react-aria/interactions/src/utils.ts

```diff
@@ -118,8 +118,10 @@ export function useSyntheticBlurEvent<Target = Element>(onBlur: (e: ReactFocusEv
       stateRef.current.observer = new MutationObserver(() => {
         if (stateRef.current.isFocused && target.disabled) {
           stateRef.current.observer.disconnect();
-          target.dispatchEvent(new FocusEvent('blur'));
-          target.dispatchEvent(new FocusEvent('focusout', {bubbles: true}));
+          // added relatedTarget because we have checks which specifically look for that property in order to determine if focus was lost to the body or intentionally moved with in the scope
```

Comment:
I think the comment is unnecessary because all "real" focus/blur events have a relatedTarget, so we should have always been adding it on our synthetic events to correctly simulate a real event.

Reply:
I'm trying to add clarity to this code so we don't have to use git blame and not recall why the change was made, like the conversation about the useEffect/useLayoutEffect above.